### ISSUE TITLE
[$15]Toggle button labeling is difficult to understand.  Reword.  Also, Dev toggle button does not repeat when you turn the butoon on and off.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3244]
**Describe the bug**
Toggle button labeling is difficult to understand.  Reword.  Also, Dev toggle button does not repeat when you turn the butoon on and off.

**To Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 
2. Ensure the screen readr is switched on. 
3. Tab till you get to the toggle buttons

**Expected behavior**

Following 3 toggle buttons should be announced as:
a. Design toggle button pressed On/Off
b. Developmenttoggle button pressed On/Off
c. Data Science toggle button pressed On/Off

Also, dev toggle should repeat on/off when it is pressed

**Actual behavior**
Text is not appropriate and Dev toggle button on/off is not being repeated by the screenreader

**Additional context**

Type: Challenge Listing Navigation

Page: https://www.topcoder.com/challenges

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099917 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099917 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099917 has been updated - it has been assigned to nahidshahin.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by nahidshahin** [comment]
PR https://github.com/topcoder-platform/community-app/pull/3254

**Comment by tosha5252** [comment]
Toggle button does not switch off or on when pressing enter or space...

Screen-reader announced "toggle button, press checked"...has not been re-worded.

Win10/Chrome/NVDA

**Comment by nahidshahin** [comment]
@tosha5252 I don't understand the problem

In this ticket I only changed the labels.
So the screen-reader should announce "Design toggle button pressed On" if design toggle was pressed.

**Comment by veshu** [comment]
@nahidshahin Yes  make toggle from keyboard too.

**Comment by nahidshahin** [comment]
@veshu I just checked https://www.topcoder.com/challenges, toggle by hitting `enter` key is working. Can you please check?

**Comment by veshu** [comment]
@tosha5252 can you please reverify clearing cache.

cc @drasticdpk @lakshmiathreya

**Comment by nahidshahin** [comment]
@tosha5252 any update?

**Comment by tosha5252** [comment]
Screen reader is easy to understand and each toggle button switches on and off when pressing enter.

**Comment by nahidshahin** [comment]
@veshu can you please process the payment?

**Comment by veshu** [comment]
@nahidshahin We need to verify on production as well before releasing the payment. Once we verified we will process the payment. Thanks!

**Comment by drasticdpk** [comment]
@tosha5252  can you please verify it?

**Comment by tosha5252** [comment]
@drasticdpk...Screen reader is easy to understand and each toggle button switches on and off when pressing enter in prod env

passes prod qa in Win10/Chrome

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by nahidshahin** [comment]
@veshu can you please process the payment?

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30099917<br/><br/>```This is an automated message for crazyk via Topcoder X```
